### PR TITLE
gitignore update

### PR DESCRIPTION
**Pull request by Kostegan**

### CODE REVIEW

File: tasks/teach/src/main/resources/Task.txt [hunk 1]
```diff
@@ -344,5 +344,5 @@ lesson 47
 2. Написать класс Period, который выдает интервал между двумя датами (заданы в миллисекундах). Интервал - читаемый для человека. Например, 1 year 5 days and 30 seconds
 
 
-
-
+lesson 48
```
Comment (Kostegan) [inline tasks/teach/src/main/resources/Task.txt:347]: what is the line?

Comment (Kostegan) [inline tasks/teach/src/main/resources/Task.txt:347]: it is ok line

Comment (vbogdanovsky) [inline tasks/teach/src/main/resources/Task.txt:347]: I think too

**Comment by vbogdanovsky** [comment]
:+1:

**Pull request merged**